### PR TITLE
Update Set-CsCallQueue.md

### PR DESCRIPTION
**Pull request by CLYVR**
Adding new AuthorizedUsers parameter

### CODE REVIEW

**Comment by CLYVR** [comment]
@get-itips @dariomws @tseward @tiburd Please and thanks

**Review by dariomws — APPROVED** [review]

File: skype/skype-ps/skype/New-CsCallQueue.md [hunk 1]
```diff
@@ -528,6 +528,22 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -AuthorizedUsers
+This is a list of user GUIDs who are authorized to make changes to this call queue.  The users must also have a TeamsVoiceApplications policy assigned.  The GUID should contain 32 digits with 4 dashes (xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx).
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/New-CsCallQueue.md:532]:
```suggestion
This is a list of GUIDs of users who are authorized to make changes to this call queue.  The users must also have a TeamsVoiceApplications policy assigned.  The GUID should contain 32 digits with four dashes (xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx). 
```

File: skype/skype-ps/skype/Set-CsCallQueue.md [hunk 2]
```diff
@@ -540,6 +540,22 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -AuthorizedUsers
+This is a list of user GUIDs who are authorized to make changes to this call queue.  The users must also have a TeamsVoiceApplications policy assigned.  The GUID should contain 32 digits with 4 dashes (xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx).
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsCallQueue.md:544]:
```suggestion
This is a list of GUIDs of users who are authorized to make changes to this call queue.  The users must also have a TeamsVoiceApplications policy assigned.  The GUID should contain 32 digits with four dashes (xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx). 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @CLYVR, kindly commit the suggested changes. Many thanks.

File: skype/skype-ps/skype/New-CsCallQueue.md (hunk 1 above)
Comment (tiburd) [inline skype/skype-ps/skype/New-CsCallQueue.md:532]:
```suggestion
This is a list of GUIDs for users who are authorized to make changes to this call queue.  The users must also have a TeamsVoiceApplications policy assigned.  The GUID should contain 32 digits with 4 dashes (xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx). 
```

File: skype/skype-ps/skype/Set-CsCallQueue.md (hunk 2 above)
Comment (tiburd) [inline skype/skype-ps/skype/Set-CsCallQueue.md:544]:
```suggestion
This is a list of GUIDs for users who are authorized to make changes to this call queue.  The users must also have a TeamsVoiceApplications policy assigned.  The GUID should contain 32 digits with 4 dashes (xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx). 
```

File: skype/skype-ps/skype/New-CsCallQueue.md [hunk 3]
```diff
@@ -528,6 +528,22 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -AuthorizedUsers
+This is a list of GUIDs for users who are authorized to make changes to this call queue. The users must also have a TeamsVoiceApplications policy assigned.  The GUID should contain 32 digits with 4 dashes (xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx).
```
Comment (tiburd) [inline skype/skype-ps/skype/New-CsCallQueue.md:532]:
```suggestion
This is a list of GUIDs for users who are authorized to make changes to this call queue. The users must also have a TeamsVoiceApplications policy assigned. The GUID should contain 32 digits with 4 dashes (xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx). 
```

File: skype/skype-ps/skype/Set-CsCallQueue.md [hunk 4]
```diff
@@ -540,6 +540,22 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -AuthorizedUsers
+This is a list of GUIDs for users who are authorized to make changes to this call queue. The users must also have a TeamsVoiceApplications policy assigned.  The GUID should contain 32 digits with 4 dashes (xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx).
```
Comment (tiburd) [inline skype/skype-ps/skype/Set-CsCallQueue.md:544]:
```suggestion
This is a list of GUIDs for users who are authorized to make changes to this call queue. The users must also have a TeamsVoiceApplications policy assigned. The GUID should contain 32 digits with 4 dashes (xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx). 
```

**Comment by tiburd** [comment]
> Copy review - @CLYVR, kindly commit the suggested changes. Many thanks.

@CLYVR @JohanFreelancer9 
Thanks for the edit pass, Johan - I committed slightly adjusted edits based on your suggestions.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Comment by CLYVR** [comment]
@tiburd Is there anything more needed from me on this or can this be merged?  Thanks!

**Comment by tiburd** [comment]
@CLYVR 
Merging now. Sorry this fell through the cracks around the time I switched to a new device and Outlook format.....

**Pull request merged**